### ISSUE TITLE
Bug in Histogram?

### ISSUE THREAD

**Issue by rnugent3** [HydrologicEngineeringCenter/fda-statistics#73]
I think there is a bug at this line:

https://github.com/HydrologicEngineeringCenter/fda-statistics/blob/bd12ee62c5753b70343f5479f77f64b70256ebb4/Statistics/Histograms/Histogram.cs#L482

Should the argument of PDF be the qvalue?

**Comment by HenryGeorgist** [comment]
yes. im not sure how that bug got in, but i fixed it in threadsafehistogram already. if you want to fix it you can
